src/Tests.Microsoft.Security.Utilities.Core/IdentifiableSecretsTests.cs

```diff
@@ -334,6 +336,11 @@ public void IdentifiableSecrets_TryValidateCommonAnnotatedKeyBase64EncodedChecks
                     bool result = IdentifiableSecrets.TryValidateCommonAnnotatedKey(keyBytes, signature);
                     result.Should().BeTrue(because: $"{key}' should validate using 'TryValidateCommonAnnotatedKey' with its original checksum");
 
+                    string differentSignature = GetRandomSignature();
+                    differentSignature.Should().NotBe(signature, because: "it is random.");
```

Comment:
>. [](http://example.com/codeflow?start=87&length=1)

NIT: trailing periods typically don't render well in these API, so we don't add them.